test/helpers/kubectl.go

```diff
@@ -1965,16 +1965,12 @@ func (kub *Kubectl) GetCiliumPodOnNode(namespace string, node string) (string, e
 }
 
 func (kub *Kubectl) ciliumPreFlightCheck() error {
-	switch GetCurrentIntegration() {
-	case CIIntegrationFlannel:
-	default:
-		err := kub.ciliumStatusPreFlightCheck()
-		if err != nil {
-			return fmt.Errorf("status is unhealthy: %s", err)
-		}
+	err := kub.ciliumStatusPreFlightCheck()
```

Comment:
Won't this fail if running in flannel mode then? Or better yet, how come `cilium status` is successful if there isn't any cilium-health endpoint running?

Reply:
It won't fail, this is `cilium status` which does not account for anything in `cilium-health`.

It doesn't make sense for `cilium status` to fail if anything with `cilium-health` is in a bad state, for the local endpoint there's a controller to handle failures, while for the cluster state we would descend into madness if any connectivity blip across the cluster caused the local cilium instance to fail out.